mermaid/__main__.py

```diff
@@ -43,12 +44,36 @@ def __init__(
 
         Parameters:
             graph (Graph): The Graph object containing the Mermaid diagram script.
+            width (Optional[int]): The width of the SVG image.
+            height (Optional[int]): The height of the SVG image.
+            scale (Optional[int]): The scale of the SVG image.
+            position (Union[Position, str]): The position of the node in the Mermaid diagram.
         """

```

Comment:
here you can add assertion to validate the scale
```python
# Validate the scale parameter if it is provided and validate the width and height parameters if scale is provided
        if scale:
            assert 1 <= scale <= 3, "Scale must be between 1 and 3"
            assert any([width, height]), "One or both of width and height must be provided"
 ```
 and also  in the docs  of mermaid.ink the `scale` can be float between 1 and 3

Reply:
Added, good catch